### ISSUE TITLE
🐛 [BUG] - Error: Cannot find module '~/errors'

### ISSUE THREAD

**Issue by FezVrasta** [carlocorradini/graphql-auth-directive#187]
### Description

Error: Cannot find module '~/errors'
Require stack:
- .../graphql/node_modules/graphql-auth-directive/build/buildAuthDirective.js
- .../graphql/node_modules/graphql-auth-directive/build/index.js

### Reproduction URL

N/A

### Reproduction steps

```sh
This happens as soon you import the file, you can see the transpiled module is using `~` in the imports, but that's not a valid path of course.
```

**Comment by carlocorradini** [comment]
Wow, nice catch! Thanks @FezVrasta 

Strange... I think the CI messed up something :)

**Comment by carlocorradini** [comment]
@FezVrasta Commit https://github.com/carlocorradini/graphql-auth-directive/commit/c328531b5d25711d4234ec591c0b51bb661dac68 should fix the problem.
I'll push a new tag as soon as possible.

**Comment by FezVrasta** [comment]
Thanks!

**Comment by carlocorradini** [comment]
Leave this open until I'll push a new tag :)

**Comment by carlocorradini** [comment]
@FezVrasta Fixed in `v5.0.1` 🥳
Ciao 👋
